Material.Blazor/Components/Dialog/MBDialog.razor.cs

```diff
@@ -65,18 +61,11 @@ public partial class MBDialog : ComponentFoundation, IMBDialog
         [Parameter] public bool OverflowVisible { get; set; } = false;
 
 
-        /// <summary>
-        /// True once the dialog has instantiated components for the first time.
-        /// </summary>
-        bool IMBDialog.DialogHasInstantiated => dialogHasInstantiated;
-
-
         private ElementReference DialogElem { get; set; }
         private bool HasBody => Body != null;
         private bool HasButtons => Buttons != null;
         private bool HasCustomHeader => CustomHeader != null;
         private bool HasTitle => !string.IsNullOrWhiteSpace(Title);
-        private List<DialogChildComponent> LayoutChildren { get; set; } = new List<DialogChildComponent>();
```

Comment:
The child components themselves wait for the dialog to be opened, so no need to list them here